{{cookiecutter.__src_folder_name}}/infra/core/host/appservice.bicep

```diff
@@ -61,9 +59,9 @@ resource appService 'Microsoft.Web/sites@2022-03-01' = {
         allowedOrigins: union([ 'https://portal.azure.com', 'https://ms.portal.azure.com' ], allowedOrigins)
       }
       // Diagnostic logging
-      detailedErrorLoggingEnabled: enableDiagnosticLogging
-      httpLoggingEnabled: enableDiagnosticLogging
-      requestTracingEnabled: enableDiagnosticLogging
+      detailedErrorLoggingEnabled: true
```

Comment:
Hm, will you be suggesting this change for azure-dev? Or it was needed here for some reason, you couldnt pass in true? Ah you're trying to make template-analyzer happy, I guess?